### DISCUSSION TITLE
Is it possible to block youtube except  for a  specific playlist of my choise?

### DISCUSSION THREAD

**Discussion by Sorion-64** [proginosko/LeechBlockNG#409] [Q&A] [answered]
Youtube is one of my biggest distractions so I want to block it on my work computer but I want to leave  access  to a specific youtube playlist that has helpful videos, is it possible to do this?

**Comment by smayer97** [comment, marked as answer]
Yes... I just set up this exact same scenario last week....every playlist has a code that is visible in the URL. 
Block set
youtube.com
+youtube.com/\*[playlist code]\* 
without the [ ] brackets; you do not have to include the entire code either, just a significant part. You can ID the code by hovering over each video in the playlist and it will become obvious.

If you click on the videos in the playlist, they will also play, as long as you always access those videos from the playlist, because the play. Any other way, and the video will be blocked.

**Reply by ls7i01zblacj0** [reply]
Thanks! I think this might also work to allow all youtube playlists by the same creator by indicating the first twelve digits after list= (I've tried this on one creator, but it seems to work on all of their playlists so far)

**Comment by SuxinL** [comment]
I think that leechblock is a general blocker for browsers which is mainly used to block domains. To localize the problem to a group pages of certain site, especially someone like Youtube with comprehensive APIs, a targeted blocking extension is better. I suggest [YTBlock](https://chromewebstore.google.com/detail/ytblock-block-any-content/nedcanggplmbbgmlpcjiafgjcpdimpea?hl=en&pli=1).
